### DISCUSSION TITLE
Found a bug!

### DISCUSSION THREAD

**Discussion by SillyIlli56** [Nerixyz/current-song2#421] [Q&A]
Hey! Just wanted to let you know that Youtube music seems to only work on OPERA browser, When trying to get it to show up on the overlay. It has to be playing the music, on an opera browser, Every other browser will not show up on the overlay, when it is trying for youtube music, HOWEVER. Every other browser will show the display properly if you play the music, ON an Opera browser.

So for example. Opera (Youtube Music) + Chrome (Overlay) = GOOD and Functioning
Opera (Overlay) + Chrome (Youtube Music) = BAD and not working.
Chrome (Youtube Music) + Firefox (Overlay) = BAD and not working.
Etc etc For all browsers I've tried, Including Edge.

So Maybe you might wanna look at it? Re-test some things, Solid program though, thanks for your hard work!

**Comment by Nerixyz** [comment]
Are you using the extension? If not, what's your configuration (`config.toml`)? By default, chrome, firefox, and edge are excluded by default:

https://github.com/Nerixyz/current-song2/blob/ab8c5d17bb64bd44783065f5c9586d512df1e33d/src/config.rs#L134-L142

I think I'll change that, since even Firefox should have basic SMTC support soon.

**Reply by SillyIlli56** [reply]
Uhm, So I don't quite know what you mean, I've downloaded the EXE and done all the steps, I did notice things were excluded by default, and I changed it to "mode" "include" and it didnt fix that problem, No matter what Opera playing the music was the only solution, 

I'm glad you're going to take a look at it/change things though <3

**Reply by Nerixyz** [reply]
Hmm, I just tested it on Chrome and Firefox with the following config for GSMTC and it worked fine:

```toml
[modules.gsmtc.filter]
mode = "Include"
items = ["Spotify.exe", "firefox.exe", "chrome.exe"]
```

**Comment by SillyIlli56** [comment]
I just tested this and no, it still doesnt work with music youtube coming
from firefox for me, Opera instantly works. No idea.
